config.py

```diff
@@ -67,15 +67,15 @@ class Config:
                     'allowed_domains': ['www.ebi.ac.uk'],
                     'allowed_request_domains' : ['targetvalidation.org', 'alpha.targetvalidation.org','beta.targetvalidation.org','localhost', '127.0.0.1'],
                     }
-    REDIS_SERVER ='/tmp/api_redis.db'
+    REDIS_SERVER =os.getenv('REDIS_SERVER', '/tmp/api_redis.db')
```

Comment:
this should be called `REDIS_SERVER_PATH` since it is a path to the persisted file